.github/workflows/build-and-publish.yml

```diff
@@ -11,13 +11,14 @@ jobs:
   cd-job:
     name: Continuous Delivery
     runs-on: ubuntu-24.04
+    permissions:
```

Comment:
mostly PTB updates, but some hand-crafted changes for the ITDE's separate ymls. see fast-tests, get-exasol-versions, periodic-validation, publish-docker-runner, test-db-version*